doc/whats_new/v0.24.rst

```diff
@@ -367,6 +367,17 @@ Changelog
   :meth:`tree.DecisionTreeRegressor.fit`, and has not effect.
   :pr:`17614` by :user:`Juan Carlos Alfaro Jiménez <alfaro96>`.
 
+- |Fix| A fix to allow a binary serialized model to be successfully loaded on a machine
+  with different endianness.
+  :pr:`17644` by :user:`Qi Zhang <qzhang90>`.
+
+:mod:`sklearn.utils`
+.........................
+
+- |Fix| Fix :func:`utils.estimator_checks.check_estimator` so that all test
+  cases support the `binary_only` estimator tag.
+  :pr:`17812` by :user:`Bruno Charron <brcharron>`.
```

Comment:
This line is due to a merge conflict and should be removed.

Reply:
@rth do you mean to remove the following section?
```
:mod:`sklearn.utils`
.........................

- |Fix| Fix :func:`utils.estimator_checks.check_estimator` so that all test
  cases support the `binary_only` estimator tag.
  :pr:`17812` by :user:`Bruno Charron <brcharron>`.
```

Reply:
Yes, if you look at the PR diff on Github it should only contain the code you modified and nothing else.